journal/tasks.py

```diff
@@ -134,3 +134,19 @@ def fetch_and_process_journal_logo(self, collection_acron3, domain, journal_acro
                 "domain": domain,
             },
         )
+
+
+@celery_app.task(bind=True)
```

Comment:
@samuelveigarangel estas tarefas não são necessárias, podem ser removidas. No entanto, as lógicas podem ser incorporadas na função fetch_and_process... com o uso de parâmetros que indicam que é para forçar a atualização  e pular a coleta etc